eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/InsertWebHookConfigHandler.java

```diff
@@ -76,24 +74,18 @@ public InsertWebHookConfigHandler(WebHookConfigOperation operation) {
     /**
      * Handles requests by adding a WebHook configuration.
      *
-     * @param httpExchange the exchange containing the request from the client and used to send the response
-     * @throws IOException if an I/O error occurs while handling the request
+     * @throws Exception if an I/O error occurs while handling the request
      */
-    @Override
-    public void handle(HttpExchange httpExchange) throws IOException {
-        NetUtils.sendSuccessResponseHeaders(httpExchange);
 
-        // Resolve to WebHookConfig
-        String requestBody = NetUtils.parsePostBody(httpExchange);
-        WebHookConfig webHookConfig = JsonUtils.parseObject(requestBody, WebHookConfig.class);
+    @Override
+    public void handle(HttpRequest httpRequest, ChannelHandlerContext ctx) throws Exception {
+        Map<String, Object> body = parseHttpRequestBody(httpRequest);
+        Objects.requireNonNull(body, "body can not be null");
+        WebHookConfig webHookConfig = JsonUtils.mapToObject(body, WebHookConfig.class);
+        // Add the WebHookConfig if no existing duplicate configuration is found
+        Integer code = operation.insertWebHookConfig(webHookConfig); // operating result
+        String result = 1 == code ? "insertWebHookConfig Succeed!" : "insertWebHookConfig Failed!";
+        write(ctx, result.getBytes(Constants.DEFAULT_CHARSET), HttpHeaderValues.TEXT_HTML);
 
-        try (OutputStream out = httpExchange.getResponseBody()) {
-            // Add the WebHookConfig if no existing duplicate configuration is found
-            Integer code = operation.insertWebHookConfig(webHookConfig); // operating result
-            String result = 1 == code ? "insertWebHookConfig Succeed!" : "insertWebHookConfig Failed!";
-            out.write(result.getBytes(Constants.DEFAULT_CHARSET));
-        } catch (Exception e) {
-            log.error("get WebHookConfigOperation implementation Failed.", e);
-        }
     }
```

Comment:
`HttpHeaderValues.TEXT_HTML` param can be wrapped in `AbstractHttpHandler`.

Redundant line L89 & L79.

Reply:
noted，please review